bootstrap/src/main/java/io/jenkins/jenkinsfile/runner/bootstrap/Bootstrap.java

```diff
@@ -109,6 +119,15 @@ private void postConstruct() throws IOException {
                 installPlugin(shortname, version);
             }
         }
+
+        if (this.runWorkspace != null){
+            if (System.getProperty("jenkins.model.Jenkins.workspacesDir") != null) {
```

Comment:
You have this property used multiple times. Want to refactor into a constant?

Reply:
no. but i will